.github/workflows/push.yml

```diff
@@ -63,24 +63,14 @@ jobs:
         uses: ./.github/actions/verify-generated-files
   LINUX_X32:
     name: LINUX_X32_DEBUG_ZTS
-    runs-on: ubuntu-latest
-    container:
-      image: ubuntu:20.04
```

Comment:
this is needed, otherwise `apt` step throws:
```
 + dpkg --add-architecture i386
dpkg: error: unable to create new file '/var/lib/dpkg/arch-new': Permission denied
```